staging/src/k8s.io/kubectl/pkg/kuberc/kuberc.go

```diff
@@ -0,0 +1,458 @@
+/*
+Copyright 2024 The Kubernetes Authors.
```

Comment:
nit, apply to all new files:
```suggestion
Copyright 2025 The Kubernetes Authors.
```

Reply:
Updated.